api/v1/main/service/meetup_service.py

```diff
@@ -44,19 +40,26 @@ def save_new_meetup(meetup_data):
     images = meetup_data["images"]
     Tags = meetup_data["Tags"]
     createdOn = datetime.datetime.utcnow()
-    happeningOn = create_future_date(date_data=int(meetup_data["happeningOn"]))
+    happeningOn = create_future_date(date_data=meetup_data["happeningOn"])
 
     meetup = get_meetup_by_topic(topic=topic)
     is_admin = UserAuth.get_admin
 
+    if happeningOn == "":
```

Comment:
You can also check if provided value is an integer here

Reply:
Noted.